roles/omero_server/tasks/omero-systemd.yml

```diff
@@ -9,7 +9,7 @@
     src: systemd-system-omero-server-service.j2
     mode: "0644"
   notify:
-    - Reload systemd # handler from ome.common
+    - reload systemd # noqa: name[casing]
```

Comment:
The handler used here is coming from the (third party) `ome.common` role.

Reply:
Ah that might have been my bad fighting with Ansible lint